<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetForm</class>
 <widget class="QWidget" name="BudgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Max Budget:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="maxBudgetLabel">
       <property name="text">
        <string>$ 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Budget:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="budgetBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="prefix">
        <string>$ </string>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="budgetSlider">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>budgetSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>budgetBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>budgetBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>budgetSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
